gsw_oceanographic_toolbox.c

```diff
@@ -3242,11 +3242,11 @@ elemental subroutine gsw_frazil_properties_potential_poly (sa_bulk, &
 !  Calculates the mass fraction of ice (mass of ice divided by mass of ice
 !  plus seawater), w_Ih_final, which results from given values of the bulk
 !  Absolute Salinity, SA_bulk, bulk potential enthalpy, h_pot_bulk,
-!  occuring at pressure p.  The final equilibrium values of Absolute
+!  occurring at pressure p.  The final equilibrium values of Absolute
 !  Salinity, SA_final, and Conservative Temperature, CT_final, of the
 !  interstitial seawater phase are also returned.  This code assumes that
 !  there is no dissolved air in the seawater (that is, saturation_fraction
-!  is assumed to be zero thoughout the code).
+!  is assumed to be zero thorughout the code).
```

Comment:
Still a typo.

Reply:
I wonder why codespell suggested that change :-/ I probably added that to its dict.